src/test/standardTest.ts

```diff
@@ -70,7 +98,8 @@ async function start() {
         launchArgs: baseLaunchArgs
             .concat([workspacePath])
             .concat(channel === 'insiders' ? ['--enable-proposed-api'] : [])
-            .concat(['--timeout', '5000']),
+            .concat(['--timeout', '5000'])
```

Comment:
Changed how we start VS Code tests, to ensure we disable start page. See `createSettigns`.